testing/sdk_metrics.py

```diff
@@ -24,6 +25,22 @@ def get_scheduler_metrics(service_name, timeout_seconds=15 * 60):
     return sdk_cmd.service_request("GET", service_name, "/v1/metrics").json()
 
 
+def get_failure_metrics(service_name: str) -> typing.Dict:
+    history = sdk_cmd.history_request().json()
+    service_history = [h for h in history["frameworks"] if h["name"] == service_name]
+    if not service_history:
```

Comment:
Is not finding the service an acceptable state?

Reply:
I experienced that the entry in the history server might not have been created yet when querying for the plan status. Thus I am not sure how to deal with the not found case.